tests/integrationv2/configuration.py

```diff
@@ -34,6 +34,12 @@
     Curves.P384,
 ]
 
+# List of keyshares for Hello Retr Requests test.
+HRR_CLIENT_KEYSHARES = [
```

Comment:
This belongs in the HRR test, not in the framework.

Reply:
will move it :)